spec/rubocop/cop/rspec/factory_bot/attribute_defined_statically_spec.rb

```diff
@@ -1,22 +1,57 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::RSpec::FactoryBot::AttributeDefinedStatically do
+  def inspected_source_filename
+    'spec/factories.rb'
+  end
+
   it 'registers an offense for offending code' do
```

Comment:
The changes below this comment can be made separately and are extracted to #1064.